src/main/java/javax/validation/ClockProvider.java

```diff
@@ -0,0 +1,42 @@
+/*
+* Copyright 2009-2012, Red Hat, Inc. and/or its affiliates, and individual contributors
```

Comment:
I've filed [BVAL-547](https://hibernate.atlassian.net/browse/BVAL-547) for the headers. We can use the simpler ones.